src/components/Login/Login.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './Login.scss';
+
+class Login extends Component {
```

Comment:
We can always build this as a class based component initially and refactor it to use hooks since that's good practice for us.